lib/tesla.ex

```diff
@@ -182,16 +185,18 @@ defmodule Tesla.Adapter do
   @callback call(env :: Tesla.Env.t(), options :: any) :: Tesla.Env.result()
 
   @doc """
-  Helper function that merges all adapter options
+  Helper function that merges all adapter options.
 
   ## Params:
-    * `defaults` (optional) - useful to override lower level http client default configuration
-    * `env` - `Tesla.Env` struct
-    * `opts` - options provided to `Tesla.Builder.adapter/2` macro
+
+  - `:defaults` (optional) - useful to override lower level http client default configuration
+  - `:env` - `Tesla.Env` struct
+  - `:opts` - options provided to `Tesla.Builder.adapter/2` macro
```

Comment:
Same here, positional args instead of keyword args.

Reply:
Am I reading the incorrect diff or this still has `:` ? 🤔

Reply:
My bad! updated